.github/workflows/node.js.yml

```diff
@@ -19,14 +19,25 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Cache node modules
+
+      - name: Cache node_modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        with:
+          # If no package-locks or the main package.json have changed, it should be safe to restore all node_modules as they were
+          # Changing the main package.json may change the postinstall scripts, so it should be unchanged to cache install results
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/package-lock.json', 'package.json') }}
```

Comment:
Use the Node - lerna example: https://github.com/actions/cache/blob/main/examples.md#node---lerna

Reply:
Effectively doing that, except since I'm skipping `npm ci` on cache hit I'm also hashing the main package.json file.

If we were to change the lerna flags on install/bootstrap and we weren't hashing that file, then it would be a cache hit and skip the install with different flags.